src/tools/post_processor/post_processor/dwarf_string_patcher.cc

```diff
@@ -549,6 +567,11 @@ ReturnCode DWARFStringPatcher::ProcessLineInfoData(
                            data_length,
                            swap_byte_ordering);
 
+  // TODO(bkase): Actually use the full prefix map
+  auto iter = prefix_map_.begin();
+  std::string old_prefix_ = iter->first;
```

Comment:
The other parts of the code are using `std::string` fully written out so as much as auto would make life more amazing, I should probably keep it as `std::string` to be in-line with the rest of the style.

Giving a precise type for the iterator is super noisy though, so there I make an exception